src/sentry/api/endpoints/client_state.py

```diff
@@ -18,6 +18,32 @@
 }
 
 
+class ClientStateListEndpoint(OrganizationEndpoint):
+    private = True
+
+    def __init__(self, **options) -> None:
+        cluster_key = getattr(settings, "SENTRY_CLIENT_STATE_REDIS_CLUSTER", "default")
+        self.client = redis.redis_clusters.get(cluster_key)
+        super().__init__(**options)
+
+    def get_key(self, organization, category, user):
```

Comment:
@Neo-Zhixing is it possible to centralize this key generation function? Looks like it's duplicated from `ClientStateEndpoint.get_key()` as well.

It'll be easier to update these keys from just one place; especially when I'll add a new key for the performance checklist onboarding soon 😄